src/components/TodoList/TodoList.tsx

```diff
@@ -1,3 +1,25 @@
+import React from 'react';
+import { TodoInfo } from '../TodoInfo';
+import './TodoList.scss';
+
+interface Props {
+  preparedTodos: Todo[];
+}
+
+export const TodoList: React.FC<Props> = ({ preparedTodos }) => {
+  return (
```

Comment:
implicit return begs to be used
```suggestion
export const TodoList: React.FC<Props> = ({ preparedTodos }) => (
```